_pages/man-camp-its-gonna-be-hard.html

```diff
@@ -0,0 +1,98 @@
+---
+layout: container-fluid
+title: MCX Warning, it's Gonna be Hard
+permalink: /mancamp/man-camp-its-gonna-be-hard/
+---
+
+<div class="row">
+  <div class="jumbotron" style="background-image: url('//crds-cms-uploads.imgix.net/content/images/crossroads-man-camp-mcx1-lg.jpg');" data-optimize-bg-img>
+    <div class="container-fluid">
+      <div class="row push-ends soft-ends">
+        <div class="col-md-offset-2 col-md-8 push-ends soft-sides">
+          <h1 class="font-family-condensed-extra text-uppercase" style="line-height: .9em;">MCX Warning,<br>it's Gonna be Hard</h1>
+          <span class="tagline" style="color: #fff;"><em>by Judd Watkins</em> </span>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container push-bottom">
+    <div class="row">
+      <div class="col-sm-offset-2 col-sm-8 soft-sides">
```

Comment:
you can just have one of these divs with all the `p`s inside. That would clean things up a bit.